src/Concerns/PerformsCharges.php

```diff
@@ -23,7 +24,6 @@ trait PerformsCharges
     public function charge($amount, $paymentMethod, array $options = [])
     {
         $options = array_merge([
-            'automatic_tax' => $this->automaticTaxPayload(),
```

Comment:
I removed this because it's not possible to use automatic tax with payment intents atm it seems.